lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -0,0 +1,69 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+"use strict";
+
+(function (intrinsic) {
+    var platform = intrinsic.JsBuiltIn;
+
+    let FunctionsEnum = {
+        ArrayIndexOf: { className: "Array", methodName: "indexOf", argumentsCount: 1 },
+    };
+
+    platform.registerFunction(FunctionsEnum.ArrayIndexOf, function (searchElement, fromIndex) {
+        // https://www.ecma-international.org/ecma-262/7.0/index.html#sec-array.prototype.indexof
```

Comment:
change to "ECMAScript 2017 #sec-array.prototype.indexof" LGTM.

---
In reply to: [116853125](https://github.com/Microsoft/ChakraCore/pull/2898#discussion_r116853125) [](ancestors = 116853125)